code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -71,13 +70,11 @@
 	new /obj/item/stack/cable_coil/random(src, 30)
 	update_icon()
 
-/obj/item/storage/belt/utility/full/multitool/New()
-	..()
+/obj/item/storage/belt/utility/full/multitool/populate_contents()
 	new /obj/item/multitool(src)
 	update_icon()
```

Comment:
will not be needed if you do the parent call in a smart way

Reply:
Making the multitool spawn first would avoid the need for two `update_icon()`s, but personally I think it'd feel a bit odd to change the positioning of the tools.